pkg/wavefront/api.go

```diff
@@ -0,0 +1,19 @@
+package wavefront
+
+import "context"
+
+type Wavefront struct {
+
+}
+
+//CreateOrUpdateWavefrontAlert creates/update a wavefront alert
+func (w *Wavefront)	CreateOrUpdateWavefrontAlert(ctx context.Context, input AlertInput) (output AlertOutput, err error) {
```

Comment:
Assuming wavefront create/update alert APIs are the same?

Reply:
Yeah- That is the assumption for now